scripts/screeps_rl_live_dashboard.py

```
@@ -0,0 +1,806 @@
+    summary = subparsers.add_parser("summary", help="print the live dashboard summary JSON")
+    add_common_args(summary)
+
+    health = subparsers.add_parser("healthcheck", help="check a running dashboard /healthz endpoint")
+    health.add_argument("--url", default=DEFAULT_HEALTHCHECK_URL, help="Health URL.")
+    health.add_argument("--timeout", type=float, default=5.0, help="HTTP timeout seconds.")
+    return parser
+
+
+def run_healthcheck(url: str, timeout: float) -> int:
+    try:
+        with urllib.request.urlopen(url, timeout=timeout) as response:
```

Ruff (0.15.13)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)